### DISCUSSION TITLE
Shift Tab key equivalent

### DISCUSSION THREAD

**Discussion by CodeNameR** [espanso/espanso#1177] [Feature requests and Ideas]
Hi

Could you please help with how I can pass "Shift+tab" key equivalent in the replace: text.
I can use \n for new line, and \t for tab, but not able to figure out how to pass Shift+tab.

Thanks,
Ritesh

**Comment by federico-terzi** [comment]
Hey,
At the moment this is not possible, what are you trying to do?

Cheers :)

**Comment by CodeNameR** [comment]
I've started using [Roam Research](https://roamresearch.com/), and was looking to build some text expanders on Windows to use at templates. Shift+Tab is kind of a key thing to get the templates right.

**Comment by federico-terzi** [comment]
All right, I see. Unfortunately, there is no way with the current architecture to do what you want to do, but I'll see if I can include this functionality in a future update!

**Comment by AucaCoyan** [comment]
Possibly it would be solved #1933
